script.js

```diff
@@ -0,0 +1,58 @@
+      $(document).ready(function(){
+
+      var message= $(".messages");
+      var turn = $(".turn");
+      var table = $(".table");
+      var cells = $(".cell");
+      var reset = $(".reset");
+      var counter= 0;
+
+
+      function playerMoves(){
+       if (counter == 0){
+      $(this).text("X").addClass("x");
+      counter++;
+      } else if (counter == 1){
+      $(this).text("O").addClass("o");
+      counter--;
+    }
+  };
+
+
+    function eventListener(){
+     cells.click(playerMoves);
+
+    }
+
+    eventListener();
+
+    function disableClick(){
+     cells.click >= 2;
```

Comment:
not sure this line is working as expected. maybe in the playermoves function, you could disable `$(this)`